var/spack/repos/builtin/packages/glfw/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Glfw(CMakePackage):
+    """GLFW is an Open Source, multi-platform library for
+    OpenGL, OpenGL ES and Vulkan development on the desktop. It
```

Comment:
Does this package install an OpenGL library? If so, you can add:
```python
provides('gl')
```
We currently have `mesa` and `opengl` as the only packages in Spack that provide the `libGL` library.

Reply:
glfw does not provide libGL